common-app-lib/src/cmr/common_app/services/search.clj

```diff
@@ -36,6 +47,17 @@
     (mem-cache/create-in-memory-cache :ttl {} {:time-to-live (scroll-id-cache-ttl)})
     (redis-cache/create-redis-cache {:ttl (/ (scroll-id-cache-ttl) 1000)}))))
 
+(defn create-scroll-first-page-cache
+  "Returns a single-threaded cache wrapping a fallback cache that uses a consistent cache backed by
+  Redis. This cache is used to store a map of cmr scroll-ids to the first page of results
+  in a consistent way acrosss all instances of search. This is used to support scrolling with
+  sessions intitiated with a HEAD request."
```

Comment:
Does it have to be a HEAD request - can they also do a GET to initiate a deferred scrolling session?

Reply:
I saw from the docs it works for HEAD, GET, or POST.

Reply:
I'll update the docstring.

Reply:
Updated.